pages/admin/invite/userAccountPage/[id].tsx

```diff
@@ -0,0 +1,166 @@
+      ? ""
+      : "hidden";
+  };
+  const deleteUser = async (): Promise<void> => {
+    try {
+      const response = await fetch("/api/admin/users/delete", {
+        method: "DELETE",
+        headers: { "Content-Type": "application/json" },
+        credentials: "include",
+        body: JSON.stringify({
+          id,
+        } as DeleteUserDTO),
+      });
+      if (!response.ok) {
+        throw await response.json();
+      }
+      onDelete();
+    } catch (err) {
+      throw new Error(err.message);
+    }
+  };
+
+  const acceptUser = async (): Promise<void> => {
+    try {
+      const response = await fetch(`/api/admin/users/${user?.id}`, {
+        method: "PATCH",
+        headers: { "Content-Type": "application/json" },
+        credentials: "include",
+        body: JSON.stringify(({
+          emailVerified: new Date(),
+          upstreamRoles: selectedPlayers,
+        } as unknown) as UpdateUserDTO),
+      });
+      if (!response.ok) {
+        throw await response.json();
+      }
+      onAccept();
+    } catch (err) {
+      throw new Error(err.message);
+    }
+  };
+  // const created = new Date();
+  return (
+    <DashboardLayout>
+      <div className="flex-col mx-16 mt-14">
+        <p className="text-4xl font-semibold">
+          Invitation Request for {user?.name}{" "}
+        </p>
+        <p className="mb-10">Created {user?.createdAt}</p>
+        <p className="text-3xl font-semibold mb-10">Basic Information </p>
+        <p className="text-1xl font-semibold">Name</p>
+        <p className="mb-10">{user?.name}</p>
+        <p className="text-1xl font-semibold">Email Address</p>
+        <p className="mb-10">{user?.email}</p>
+        <p className="text-1xl font-semibold">Phone number</p>
+        <p className="mb-10">{user?.phoneNumber}</p>
+        <p className="text-1xl font-semibold">Role</p>
+        <p className="mb-10">{user?.defaultRole.type}</p>
+        <p className="text-2xl font-semibold mb-10">Attached Note</p>
+
+        <div className={showCombobox()}>
+          <p className="text-2xl font-semibold"> Role Information </p>
+          <p className="text-1xl font-semibold">Linked Players</p>
+          <p className="mb-20">
+            {user && (
+              <Combobox
+                selectedPlayers={selectedPlayers}
+                setSelectedPlayers={setSelectedPlayers}
+                role={user.defaultRole.type}
+              />
+            )}
+          </p>
+        </div>
+        <div className="flex space-x-4 mb-10">
+          <div>
+            <Button
+              className="bg-red-200 hover:bg-red-200 text-danger font-bold py-2 px-4 rounded"
+              onClick={deleteUser}
+            >
+              Decline
+            </Button>
+          </div>
+          <div className="ml-4 ">
+            <div>
+              <Button
+                className="bg-green-200 hover:bg-green-200 text-success font-bold py-2 px-4 rounded"
```

Comment:
To match the mockups, this should be:

```suggestion
                className="bg-success-muted text-success font-bold py-2 px-4 rounded"
```